content/altinn-studio/reference/ux/components/Date/_index.en.md

```diff
@@ -0,0 +1,133 @@
+weight: 10 # Do not change, the components will be sorted alphabetically
+toc: true
+---
+
+## Usage
+
+The Date component allows users to add structured formatted date, with or without label
+
+### Anatomy
+
+1. **Horizontal**: A date with label structured horizontally
+    ![Date horizontal anatomy](date-horizontal.png "Date Horizontal")
+2. **Vertical**: A date with label structured vertically 
+    ![Date vertical anatomy](date-vertical.png "Date vertical")
+
+
+<!-- 
+Add the following sections if relevant:
+
+### Behavior
+
+(How the component behaves in different contexts)
+
+### Style
+
+(Visual styling (e.g. alignment, padding, dos and don'ts))
+
+### Best Practices
+
+(Industry standards, dos and don'ts)
+
+### Content guidelines
+
+(E.g. punctuation rules, standard labels, etc.)
+
+### Accessibility
+
+(Component-specific best practices for accessibility.)
+
+### Mobile
+
+(How to apply component in mobile environments.)
+
+-->
+### Related
+
+- [`Number`](../number/)
+- [`Text`](../text/)
+
+## Properties
+
+{{% notice warning %}}
+We are currently updating how we implement components, and the list of properties may not be entirely accurate.
+{{% /notice %}}
+
+| **Property**                 | **Type** | **Description**                                                                          |
+|------------------------------|----------|------------------------------------------------------------------------------------------|
+| `id`                         | string   | Unique Id string for the component                                                       |
+| `value`                      | string   | The date you want to display. Must be a ISO6801 string.                                  |
+| `textResourceBindings.title` | string   | Label of the date you want to display                                                    |
+| `format`                     | string   | A formatting string based on [Unicode Technical standard](https://www.unicode.org/reports/tr35/tr35-dates.html#Date_Field_Symbol_Table)                              |
+| `direction`                  | string   | Sets the structured direction of label and value. **Enum:** `"horizontal" \| "vertical"` |
+| `icon`                       | string   | An URL path to the image/icon                                                            |
+
+## Configuration
+
+### Add component
+
+```json{hl_lines="6-"}
+{
+  "id": "date",
+  "type": "Date",
+  "textResourceBindings": {
+    "title": "Date"
+  },
+  "value": "2022-09-27 18:00:00.000",
+  "format": "dd.MM.yyyy",
+  "direction": "horizontal",
+  "icon": "https://www.example.com/icon.png"
+},
+```
+
+### Add title and elements
+
+<br>
+
+#### `textResourceBindings.title`
+
+The title for the date is shown as a label and can be added as text directly or referenced via a text key to a [text resource](/nb/altinn-studio/reference/ux/texts/#legge-til-og-endre-tekster-i-en-app).
```

Comment:
```suggestion
Label for the date. This can be added as a hardcoded string or as a reference to a [text resource](/nb/altinn-studio/reference/ux/texts/#legge-til-og-endre-tekster-i-en-app).
```